core/org.wso2.carbon.server.admin/src/main/java/org/wso2/carbon/server/admin/service/ServerAdmin.java

```diff
@@ -175,8 +175,18 @@ public String getServerVersion() {
 
     public boolean restart() throws Exception {
 
-        if (Boolean.parseBoolean(
-                ServerConfiguration.getInstance().getFirstProperty(ServerConstants.DISABLE_REMOTE_RESTART))) {
+        boolean enableRestart;
+        if (String.valueOf(true)
```

Comment:
Any specific reason to use String.valueOf(true) rather than using Boolean.TRUE.toString() or just "true"?

Reply:
Wanted to use a defined property/constant instead of "true". Then, Boolean.TRUE.toString() can be used

Reply:
fixed in [f0a13c1](https://github.com/wso2/carbon-kernel/pull/3233/commits/f0a13c18165e9b844206b01b668cecf8de16d0ed)